README.md

```diff
@@ -20,16 +20,27 @@ To begin exploring the integration of Dagster and R:
    ```bash
    cd dagster-and-r
    ```
-3. **Install Dependencies**
-   Using [poetry](https://python-poetry.org/), install the package and its dependencies:
-   ```bash
-   poetry install
+3. **Install Python Dependencies**
+   # you'll need a version of python installed
+   Using uv
+   # install uv
+   # curl -LsSf https://astral.sh/uv/install.sh | sh
+   uv venv
+   source .venv/bin/activate
+   uv sync
    ```
+4. ** Install R dependencies**
+```
+   # from R
+   # if you haven't installed renv before
+   # install.packages("renv")
+   # renv::restore() 
+```
 
 4. **Set RETICULATE_PYTHON environment variable** 
 Determine the path to the python binary associated with this project's poetry environment.
    ```bash
-   poetry run
+   # from your viritual environment
    which python
    # /home/user/.cache/pypoetry/virtualenvs/dagster-and-r-kS5e8P_l-py3.10/bin/python
```

Comment:
I add the following to my `.Renviron`:

```bash
#.Renviron
RETICULATE_PYTHON=.venv/bin/python
```

We should probably update the README to include this new path after migrating to `uv`.